lang/en/settings.php

```diff
@@ -21,5 +21,13 @@
 $lang['doorkeeper-secret']   = '(Example) The Secret of your registered Doorkeeper Application.';
 $lang['doorkeeper-authurl']  = '(Example) The authorization endpoint URL of your Doorkeeper setup.';
 $lang['doorkeeper-tokenurl'] = '(Example) The access token endpoint URL of your Doorkeeper setup.';
-$lang['singleService']            = 'Login with single oAuth service only (disables local logins!)';
+$lang['custom-key']          = '(Custom) The Application ID of your registered Doorkeeper Application.';
```

Comment:
This is supposed to be custom/general oauth mechanism - it should not reference a specific rails implementation of oauth in a language string.